### PR TITLE
Mathoid 0 7 0

### PR DESCRIPTION
**Pull request by physikerwelt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,14 +23,12 @@
   },
   "dependencies": {
     "jsdom": "^6.0.0",
-    "speech-rule-engine": "*",
-    "yargs": "^3.0.0",
-    "MathJax": "https://github.com/mathjax/MathJax/tarball/mathjax-node-2.5.1"
+    "mathoid-mathjax": "git+https://github.com/physikerwelt/mathjax.git#up271"
```
Comment (physikerwelt) [inline package.json]: This needs to be updated before we merge

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
-- '4.1'
+  - '4'
+  - '5'
+  - '6'
+  - stable
```
Comment (d00rman) [inline .travis.yml]: Would be good to add Node 8 explicitly as well

File: package.json (hunk 1 above)
Comment (d00rman) [reply package.json]: [MathJax@v2.7.1](https://github.com/wikimedia/MathJax/tree/v2.7.1) has been published, so this needs to be changed now to `"^2.7.1"`

**Review by d00rman — CHANGES_REQUESTED** [review]
There are conflicts in this PR, so they need to be resolved before we can merge/publish. Also see comment in package.json

File: package.json (hunk 1 above)
Comment (physikerwelt) [reply package.json]: done

File: .travis.yml (hunk 2 above)
Comment (physikerwelt) [reply .travis.yml]: done

**Review by d00rman — APPROVED** [review]

**Comment by d00rman** [comment]
Published as [v0.7.0](https://github.com/wikimedia/MathJax-node/tree/v0.7.0)

**Pull request merged**